meshio/_mesh.py

```diff
@@ -74,13 +74,13 @@ def prune(self):
 
         new_cells = []
         new_cell_data = {}
-        for c in self.cells:
+        for block_counter, c in enumerate(self.cells):
```

Comment:
Better `zip`  it, not `enumerate`.

Reply:
Do you mean

```python
for block_counter, c in zip(range(len(self.cells)), self.cells):
```

It seems to me too verbose

Reply:
And I can't either `zip(self.cells, self.cell_data)`, since `self.cell_data` is a Dictionary.

Reply:
You're right. Let's merge this.